src/pages/HomePage.js

```diff
@@ -0,0 +1,78 @@
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { fetchData } from '../redux/data/dataSlice';
+
+const HomePage = () => {
+  const dispatch = useDispatch();
+  const countriesData = useSelector((store) => store.data);
+  const defaultContinents = [
+    {
+      continent: 'Africa',
+      countries: [],
+    },
+    {
+      continent: 'North America',
+      countries: [],
+    },
+    {
+      continent: 'South America',
+      countries: [],
+    },
+    {
+      continent: 'Antarctica',
+      countries: [],
```

Comment:
- [ ] Kindly try as much as you can to follow  these [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), including:
  - Colors (select one main color for your website).
  - Typography: font face, size and weight.
  - Layout: composition and space between elements.
- [ ] This will improve the consistency and aesthetic aspects of your design

### Your design

![image](https://user-images.githubusercontent.com/98453979/236464474-8400e336-dd9a-4ed6-986e-fc070a391c69.png)

![image](https://user-images.githubusercontent.com/98453979/236464557-dc7a3cc2-4c99-46a3-bc31-da1dca7c20b5.png)

### Checkout the original design

![image](https://user-images.githubusercontent.com/98453979/236464693-b64bf833-28e6-4414-8b87-cfe6333f1266.png)

![image](https://user-images.githubusercontent.com/98453979/236464769-c35837f7-8808-48e5-a64d-84ec2b7bde11.png)

Reply:
thanks for pointing that out Sir!

Reply:
fixed